### DISCUSSION TITLE
Disable URL displaying in the left bottom

### DISCUSSION THREAD

**Discussion by oskarjf** [MicrosoftEdge/WebView2Feedback#4845] [Q&A] [answered]
Hi

Is there some setting that I can set on WebView2 that disables the URL showing at the bottom of the browser window when hovering over links in the browser?
I'm having an issue with this staying in my WPF application when navigating away from the page with the browser component.

**Comment by mikeduglas** [comment, marked as answer]
`webView.CoreWebView2.Settings.IsStatusBarEnabled = false;`

**Reply by oskarjf** [reply]
Thanks, that solved it:)
